apps/dashboard/app/apps/nav_bar.rb

```diff
@@ -47,45 +60,48 @@ def self.nav_menu(hash_item)
         item = { apps: item }
       end
 
-      if item.fetch(:url, nil)
+      if item[:url]
         nav_link(item, menu_title, group_title)
-      elsif item.fetch(:apps, nil)
+      elsif item[:apps]
         nav_apps(item, menu_title, group_title)
-      elsif item.fetch(:profile, nil)
+      elsif item[:profile]
         nav_profile(item, menu_title, group_title)
-      elsif item.fetch(:page, nil)
+      elsif item[:page]
         nav_page(item, menu_title, group_title)
       else
-        # Update subcategory if title was provided
+        # Update subcategory if group title was provided
         group_title = item.fetch(:group, group_title)
         next nil
       end
     end.flatten.compact
 
-    OodAppGroup.new(apps: apps, title: menu_title, sort: false)
+    OodAppGroup.new(apps: apps, title: menu_title, icon_uri: menu_icon, sort: false)
   end
 
   def self.nav_link(item, category='', subcategory='')
-    OodAppLink.new(item).categorize(category: category, subcategory: subcategory)
+    link_data = item.clone
+    # Override URL with static link if there is match
+    static_url = STATIC_LINKS[item.fetch(:url).to_sym]
+    link_data[:url] = static_url if static_url
+    link_data[:new_tab] = item.fetch(:new_tab, false)
+    OodAppLink.new(link_data).categorize(category: category, subcategory: subcategory)
```

Comment:
I'm wondering if we want/need any of this. 

Some `STATIC_LINKS` are overlaps with `STATIC_TEMPLATES`.  I would say let's shove all that into a `develop` & `help` menu items into `STATIC_TEMPLATES`.

Looking at this is what gives me pause. It should be as simple as possible for the user, but now we're adding `url` and `new_tab` options.

The user shouldn't have to know or care about `url` attribute of something that we ship. 

These 2 nav config items should be equivalent. 
```yaml
nav_bar:
  - 'develop'

  - title: 'Other Develop'
     links:
       - 'restart'
       - 'docs'
       - 'products_dev'
       - 'products_usr'
```

While reviewing this to come up with that example I'm also concerned about this distinction we have between `links` and `urls`. That is going to be confusing for users.

Reply:
The change is not adding new properties like `url` and `new_tab`. The idea for this change was to support URL `aliases` for existing URLs, like `restart`, `sessions`, etc. If admins wanted to create custom links to these pages, they didn't need to know the URL. But this is not really needed as they can provide the actual URL.

As for `new_tab`, is still optional as before, I am just setting the default to `false`. It is currently `true` in the `OodAppLink` class and I think for the NavBar it makes more sense to be `false`.

I see your point that for something that we ship, it should be easier to use/configure.
Will remove the `STATIC_LINKS` logic

Will add the `develop` and `help` into the `STATIC_TEMPLATES`

> While reviewing this to come up with that example I'm also concerned about this distinction we have between links and urls. That is going to be confusing for users.

I think `url` is used in other configuration items with the same meaning (like an app `manifest`)
`links`could be renamed to make it clearer that it defines the list of items for a navigation menu.
maybe `items` or `menu_items`. Not quite sure.

Reply:
In case I miss understood, do you want to add support for this:

```yaml
  - title: 'Other Develop'
     links:
       - 'restart'
       - 'docs'
       - 'products_dev'
       - 'products_usr'
```

where a `string` could be an alias for a full navigation link.

Currently, a `string` is only used to match application tokens.

Reply:
Regarding simplicity for the 30%, I think we are there. There is more we can do, like your suggestion of using string based static links. But we can generate a copy of the current `navbar` like so:
```yaml
    nav_bar:
      - "files"
      - "jobs"
      - "clusters"
      - "interactive apps"
      - "sessions"
```
Shall I implement your string based static link idea?
```yaml
- title: "My Menu Title"
  links:
     - all_apps
     - my_sessions
     - support_ticket
     - docs
     - products_dev
     - products_usr
     - restart
```

Regarding `new_tab`, if I understood your comment correctly, it currently works as you described it. It is a pass through property to the `OodAppLink` class and not required to be provided. As you mention, only a handful of admins will set a value. But `OodAppLink` sets a default value that is `true` which I thought it would be better for navigation items to be false (to open in the same window).
But happy to remove the logic and use `OodAppLink` default.

Reply:
> Regarding new_tab, 

Forget about that. We can pick that up in some other ticket or thread. It seems related to simple configs, but not directly.

> Shall I implement your string based static link idea?

Back to the static link idea - yes, the alternative is the user has to supply the url or something right?  I guess if I rephrased the question, what would the alternative be?

Reply:
Yes, the alternative is that the user has to supply the URL.

OK - Will implement the changes.
The only static links that I can think of are: 
     - all_apps
     - my_sessions
     - support_ticket
     - docs
     - products_dev
     - products_usr
     - restart
 If you can think of another one, please let me know.

Reply:
Changes completed.

```yaml
    nav_bar:
      - title: "My Link Aliases"
        links:
          - "all_apps"
          - "sessions"
          - "support_ticket"
          - "docs"
          - "products_dev"
          - "products_usr"
          - "log_out"
          - "restart"
      - "all_apps"
      - "sessions"
      - "support_ticket"
      - "docs"
      - "products_dev"
      - "products_usr"
      - "log_out"
      - "restart"
```

<img width="1495" alt="Screenshot 2023-01-05 at 10 54 37" src="https://user-images.githubusercontent.com/17336210/210764407-f98e6cbe-d657-451b-8e01-6be781d5d818.png">

Reply:
This is looking good. I just need a second to play around with it but I suspect I won't need anymore changes. Thanks!